api/v1alpha/src/index.js

```diff
@@ -99,6 +101,8 @@ const policies = require('./policies/index');
 const accounts = require('./accounts/index');
 // Import the CDS API Admin service router
 const admin = require('./admin/index');
+// Import the CDS API Procurement service router
+const procurements = require('./procurements/index');
 
 /************************************************************
   API Endpoints
```

Comment:
I think we can remove the `router.all('*', checkIfAuthenticated);` [line](https://github.com/GoogleCloudPlatform/datashare-toolkit/pull/136/files#diff-bb5d23004b97fd9d4fb239ca7976122fL118) now that Istio provides the AuthN & AuthZ

Reply:
Will it be able to handle the regex's?

`if (req.path.startsWith('/docs/')) {
        return next();
    } else if (req.path.match(/\/projects\/.+\/accounts:register/g)) {
        // Allow unauthenticated calls to the accounts:register endpoint
        console.log(accounts:register called with headers: ${JSON.stringify(req.headers)} and body: ${JSON.stringify(req.body)});
        return next();
    } else if (req.path.match(/\/projects\/.+\/products/g)) {
        // Allow unauthenticated calls to the accounts:register endpoint
        console.log(products called with headers: ${JSON.stringify(req.headers)} and body: ${JSON.stringify(req.body)});
        return next();
    }`

Reply:
Yes. Updated accordingly:
api/v1alpha/istio-manifests/1.4/authz/allow-data-consumers-policy.tmpl.yaml
api/v1alpha/istio-manifests/1.4/authz/allow-marketplace-service-account-policy.tmpl.yaml